.github/stale.yml

```diff
@@ -9,7 +9,9 @@ onlyLabels: []
 exemptLabels:
   # This label is hardcoded on remind bot (https://probot.github.io/apps/reminders/) and is used by remind bot when
   # issue is being reminded.
-  - reminder
+  # - reminder
+  # This label is added by commenting with "/lifecycle frozen" by anyone
+  - lifecycle/frozen
```

Comment:
User need to add `/lifecycle frozen` or `lifecycle/frozen`?

Reply:
Now some issues and prs may need more time to discuss or review, add an exempt label to reduce tips.

Reply:
The format of label, I think should be `lifecycle/frozen` as the defined.

Reply:
It exists as https://github.com/kubeedge/kubeedge/labels/lifecycle%2Ffrozen.